docs/jwst/skymatch/description.rst

```diff
@@ -117,6 +117,16 @@ of sky in the images. This method cannot measure the true sky level, but
 instead provides additive corrections that can be used to equalize the signal
 between overlapping images.
 
+User-Supplied Sky Values
+-------------------------
+The ``skymatch`` step can also accept user-supplied sky values for each image.
+This is useful when sky values have been determined based on a custom workflow
+outside the pipeline. To use this feature, the user must provide a list of sky
+values matching the number of images (``skylist`` parameter) and set the
+``skymethod`` parameter to "user". Note that the skylist will be applied on a
+per-image basis irrespective of the groups defined in the association, and the
+order of the list must match the order of the input images.
```

Comment:
I'm curious whether this is the desired behavior for asn groups

Reply:
One value per input file allows the user to decide what to do with various groups. Allowing an input of one value per asn group might be slightly easier in some cases, but it's less flexible. I think no matter what, we will want to support inputting one value per filename.  And repeating values for all files in a group isn't _that_ burdensome.

Allowing both types of input (one value per group OR one value per filename) would also be possible, but it might be unnecessarily complicated.

I'm in favor of just merging this as-is, and deciding later if any additional functionality is desired.